test/scripts/boot-image

```diff
@@ -379,7 +379,7 @@ def main():
     # Keep test/scripts/imagetestlib.py:CAN_BOOT_TEST in sync as it
     # has the same checks again
     match image_type:
-        case "qcow2" | "server-qcow2":
+        case "qcow2" | "generic-qcow2" | "cloud-qcow2":
```

Comment:
This probably also needs an update to Schutzfile so that an openstack runner with kvm is selected for "generic-qcow2"